devTools/phpstan-src.neon

```diff
@@ -47,6 +47,10 @@ parameters:
         -
             path: '../src/Logger/Html/StrykerHtmlReportBuilder.php'
             message: '#return type with generic class ArrayObject does not specify its types\: TKey, TValue#'
+        -
+            message: "#^Function shell_exec is unsafe to use\\. It can return FALSE instead of throwing an exception\\. Please add 'use function Safe\\\\shell_exec;' at the beginning of the file to use the variant provided by the 'thecodingmachine/safe' library\\.$#"
```

Comment:
```suggestion
            message: "#^Function shell_exec is unsafe to use#"
```
Nobody wants a build to fail if a single letter changes in that long explanation.